examples/framework/preferences/minimalFootprint/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/normalize/css/normalize.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/core/css/fluid.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-by/by.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-yb/yb.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-bw/bw.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-wb/wb.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-lgdg/lgdg.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/preferences/css/Enactors.css" />
+        <link rel="stylesheet" type="text/css" href="css/style.css" />
+
+        <script type="text/javascript" src="js/loadScripts.js"></script>
+
+        <title>Minimal Footprint - Prefs Framework Integration</title>
+    </head>
+
+    <body class="prefsEditor-demo-theme fl-focus">
+        <div class="demo-goTo-link">
+            <a href="fullPage.html">Go To Display Preferences</a>
+        </div>
+        <main class="fl-centered">
+            <div class="flc-toc-tocContainer toc"> </div>
+
+            <script>
+
+                // Will load the array of scripts if the "fluid-ui-settings" cookie is found
+                fluid.load.lazyLoadScripts("fluid-ui-settings", [
```

Comment:
Could you add a comment here explaining how/why this code differs from the one in the docs - that is, that it is written against the broken-out dev version of the files. 
What would be very helpful is if the comment also included the exact path that the relevant build file would be generated at if the user were to run the "grunt buildDists" task - that is, the path they will find this at if they checked out Infusion as an npm module